lib/sky-pages-module-generator.js

```diff
@@ -67,6 +66,34 @@ function getSource(skyAppConfig) {
     'SkyAppViewportService'
   ];
 
+  let authTokenProvider = 'SkyAuthTokenProvider';
+
+  if (skyAppConfig.runtime.command === 'pact') {
+    runtimeProviders
+      .push(`{
+        provide: SkyPactService,
+        useClass: SkyPactService,
+        deps: [SkyAppConfig]
+      }`);
+    runtimeImports.push('SkyPactAuthTokenProvider');
+    runtimeImports.push('SkyPactService');
+    authTokenProvider = `{
+      provide: SkyAuthTokenProvider,
+      useClass: SkyPactAuthTokenProvider
+    }`;
+  }
+
+  runtimeProviders.push(authTokenProvider);
+
+  if (skyAppConfig.skyux.auth) {
```

Comment:
Looks like maybe the merge from master didn't go quite right.  This block is now duplicated around line 119.  

I'm not entirely sure yet why this PR didn't fail, but I can duplicate [your test SPA's failure](https://blackbaud.visualstudio.com/Products/_build?buildId=451766&_a=summary&tab=summary) locally.